### ISSUE TITLE
Switching screen reader accessibility modes is always going to have some weird problems

### ISSUE THREAD

**Issue by hai-zou** [microsoft/vscode#231001]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.94.1
- OS Version: win11

Steps to Reproduce:

1. `Ctrl + Shift + p` Toggle Screen Reader Accessibility Mode.
2. `Ctrl + ~` Open terminal.
3. First enter a pair of quotes, then type Chinese in the quotes, you'll find the first word missing.

![Image](https://github.com/user-attachments/assets/6c9ebffb-167b-4ca5-9d16-07559da21164)

![Image](https://github.com/user-attachments/assets/5972524e-414f-4398-ba7a-3b3a5b67c6b5)

**Comment by meganrogge** [comment]
are you saying your IME is impacted by enabling screen reader mode?

**Comment by hai-zou** [comment]
> are you saying your IME is impacted by enabling screen reader mode?

It seems so.

**Comment by meganrogge** [comment]
Ok, do you rely on using a screen reader or is this just something you noticed? I cannot reproduce fwiw.

**Comment by hai-zou** [comment]
> Ok, do you rely on using a screen reader or is this just something you noticed? I cannot reproduce fwiw.

I don't rely on it, just accidentally turned it on, causing me to have some inexplicable bugs

**Comment by meganrogge** [comment]
I don't have reports of this from screen reader users, and can't repro myself, thus closing.
